prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-access-term-enrolment/enrollee-access-term-enrolment.component.ts

```diff
@@ -41,20 +39,10 @@ export class EnrolleeAccessTermEnrolmentComponent extends AbstractComponent impl
   public ngOnInit() {
     const enrolleeId = this.route.snapshot.params.id;
     const accessTermId = this.route.snapshot.params.aid;
-    this.busy = forkJoin([
-      this.adjudicationResource.getSubmissionForAgreement(enrolleeId, accessTermId),
-      this.adjudicationResource.getEnrolleeById(enrolleeId)
-    ]).pipe(
-      map(([enrolleeSubmission, enrolment]: [HttpEnrolleeSubmission, HttpEnrollee]) =>
-        [
-            this.enrolleeSubmissionAdapterResponse(enrolleeSubmission),
-            this.enrolleeAdapterResponse(enrolment)
-        ]
-      )
-    ).subscribe(([enrolmentSubmission, enrolment]: [EnrolmentSubmission, Enrolment]) => {
-      this.enrolmentSubmission = enrolmentSubmission;
-      this.enrolment = enrolment;
-    });
+    this.busy = this.adjudicationResource.getSubmissionForAgreement(enrolleeId, accessTermId)
```

Comment:
We aren't consuming any of this data in the parent, so we can actually move this call into the child.

Reply:
This helps make the child more self contained and cleanups this fork join a little.

Reply:
looks like line 9 does have a reference: `enrolmentSubmission?.createdDate`
```HTML
<h6 class="text-green text-italic">Submitted {{ enrolmentSubmission?.createdDate | formatDate }}</h6>
```

Reply:
closing this and moving to where it was supposed to be